.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I'm not sure if we should add something specific to a single editor

Reply:
I agree, my mistake, I thought I did remove it before.

Reply:
Why do we add editor config here?